entc/gen/template/dialect/sql/query.tmpl

```diff
@@ -88,6 +92,137 @@ func ({{ $receiver }} *{{ $builder }}) sqlAll(ctx context.Context, hooks ...quer
 	return nodes, nil
 }
 
+{{/* Generate a method to eager-load each edge. */}}
+{{- range $e := $.Edges }}
+	func ({{ $receiver }} *{{ $builder }}) load{{ $e.StructField }}(ctx context.Context, query *{{ $e.Type.QueryName }}, nodes []*{{ $.Name }}, init func(*{{ $.Name }}), assign func(*{{ $.Name }}, *{{ $e.Type.Name }})) error {
+		{{- if $e.M2M }}
+			edgeids := make([]driver.Value, len(nodes))
+			byid := make(map[{{ $.ID.Type }}]*{{ $.Name }})
```

Comment:
```suggestion
			byids := make(map[{{ $.ID.Type }}]*{{ $.Name }})
```

Stay consistent

Reply:
Should be `edgeIDs` and `byIDs`

Reply:
I actually prefer `byid` as it describes the operation for getting a node by its ID and that's why this variable is used for.  I use this singular form quite often in ent and atlas. I'll keep it this way, but will change the naming to camelCase.